docs/source/upgrade.rst

```diff
@@ -0,0 +1,42 @@
+Overview
+========
+
+If you’re familiar with previous releases of Hyperledger Fabric, you’re aware
+that upgrading the nodes and channels to the latest version of Fabric is, at a
+high level, a four step process.
+
+1. Backup the ledger and MSPs.
+2. Upgrade the orderer binaries in a rolling fashion to the latest Fabric version.
```

Comment:
See FAB-3364. Can we provide more information about ledger backup and restore?

Reply:
Added a section to the Upgrade Components doc.